pygmt/datasets/load_remote_dataset.py

```diff
@@ -7,10 +7,8 @@
 from typing import Any, Literal, NamedTuple
 
 import xarray as xr
-from pygmt.clib import Session
 from pygmt.exceptions import GMTInvalidInput
-from pygmt.helpers import build_arg_list, kwargs_to_strings
-from pygmt.src import which
+from pygmt.helpers import kwargs_to_strings
```

Comment:
At line 503, `@kwargs_to_strings(region="sequence")` can be removed, since the `region` parameter, either a sequence or a string, can be directly passed to `xr.load_dataarray`.